README.md

```diff
@@ -95,13 +95,16 @@ kubectl get pkg
 #### GCP Provider Setup
 
 Set up your GCP account keyfile by following the instructions on:
-https://crossplane.io/docs/v1.0/getting-started/install-configure.html#select-provider
+https://crossplane.io/docs/v1.9/getting-started/install-configure.html#select-a-getting-started-configuration
 
 Ensure that the following roles are added to your service account:
 
 * `roles/compute.networkAdmin`
 * `roles/container.admin`
 * `roles/iam.serviceAccountUser`
+* `roles/iam.securityAdmin`
```

Comment:
what caused these extra roles/permissions to now be needed?

Reply:
@jbw976 it is required to create `ProjectIAMMember`

I mentioned here https://github.com/upbound/platform-ref-gcp/pull/22#issuecomment-1259399923

we need all that machinery to enable automated access in the form of

```
          identity:
            type: GoogleApplicationCredentials
```
for the helm ProviderConfig

Reply:
aha yes, thanks for clearing that up.  So just to tie some loose ends together (for myself), this is the new approach for getting Auth to work with GKE clusters.  This support was added to:

* provider-helm in https://github.com/crossplane-contrib/provider-helm/pull/109
* provider-kubernetes in https://github.com/crossplane-contrib/provider-kubernetes/pull/10

And that functionality generally address issues we've seen around this topic, like:

* https://github.com/crossplane-contrib/provider-gcp/issues/343
* https://github.com/crossplane-contrib/provider-gcp/issues/415